package.json

```diff
@@ -55,8 +40,41 @@
 	},
 	"husky": {
 		"hooks": {
-			"commit-msg": "commitlint -E  HUSKY_GIT_PARAMS",
+			"commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
 			"pre-commit": "pretty-quick --staged && npm run precommit-msg && npm run lint-diff"
 		}
-	}
+	},
+	"dependencies": {
+		"@commitlint/cli": "^11.0.0",
+		"@commitlint/config-conventional": "^11.0.0",
+		"@semantic-release/changelog": "^5.0.1",
+		"@semantic-release/git": "^9.0.0",
+		"eslint": "^7.9.0",
+		"eslint-config-5app": "^0.12.3",
```

Comment:
As it's a node project could you also add `eslint-plugin-node` and extend `.eslintrc` with `plugin:node/recommended`. Thats really good for catching weird things in package distributions.

As part of that you'll need a `files` section else it'll complain, something like 

```
files: ['src/','index.js'] 
```

Reply:
done